app/state.go

```diff
@@ -1194,6 +1261,15 @@ type inventoryUpdateState struct {
 	baseState
 }
 
+func NewInventoryUpdateState() State {
```

Comment:
Question: Since this is a new function, but the `inventoryUpdateState` struct is not new, where is this constructed in the existing code? We should probably change all instances to use this function now that we have it.

Reply:
right. I have found only one: https://github.com/mendersoftware/mender/pull/943/commits/f27eaffbdde042b3c89b8f1c83e797d3bc08417a#diff-0b03a1a30843453d28f38533c95445ff2cd5009a25d18c640ef66426efc044e8R93